code/modules/language/spanish.dm

```diff
@@ -0,0 +1,18 @@
+/datum/language/spanish
+	name = "Old Earth Spanish"
+	desc = "A less common Earth language that's survived the ages, thanks to being so widespread for its time."
+	speech_verb = "says"
+	whisper_verb = "whispers"
+	key = "e"
+	flags = TONGUELESS_SPEECH | LANGUAGE_HIDE_ICON_IF_UNDERSTOOD
+	default_priority = 100
+
+	icon_state = "galcom"
```

Comment:
I added a sprite but it's not showing since the PR is closed and not showing commits. I edited it into the OP. It's just a tiny mexican flag I made